test/jobs/analyze_change_job_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+require 'minitest/mock'
+
+class AnalyzeChangeJobTest < ActiveJob::TestCase
+  test 'produces change with annotations' do
+    change = changes(:page1_change_1_2)
```

Comment:
> I use the `changes` fixture. I wasn't sure whether I should use that fixture, or just make up a new pair of uuids because it appears that a new `Change` resource is created within this job.

Ah, so this job is largely responsible for creating changes, so it might be better to create a new pair of UUIDs to analyze. (Technically the job makes an annotation, which makes a relevant change if one doesn’t already exist. But most of the time this job runs, that change doesn’t exist yet. A change mainly exists as a way to tie together and summarize multiple annotations.)

Reply:
Done!